config/environment/test.php

```diff
@@ -8,7 +8,14 @@
 return array(
 
     // Disable logging
-    'Psr\Log\LoggerInterface' => DI\object('Psr\Log\NullLogger'),
+    'Psr\Log\LoggerInterface' => \DI\decorate(function ($previous, ContainerInterface $c) {
+        $enableLogging = $c->get('ini.tests.enable_logging') == 1;
```

Comment:
Is there a way to log only to file? I wonder if logging to stdout/print whether it would fail eg some system tests because the logging output would maybe appear there? (not sure....)

What could be useful as well is a command line option for `tests:run` but not sure if that makes things too complicated (be faster to just add `-vvv`) or something to the command than enabling it. Not needed though.

All good for this PR but would maybe check if logging to file only is possible? (or maybe doesn't make sense).

Can you add a developer changelog entry?

Reply:
I think both requests are doable... Will do the following:

* [ ] during tests only log to file
* [ ] set the ini config in tests:run if command line option given (I think it's useful having a separate option since -vvv will also print a lot of stuff to the screen)
* [ ] add to developer changelog